client/core/bond.go

```diff
@@ -724,6 +722,12 @@ func (c *Core) rotateBonds(ctx context.Context) {
 			c.updateAssetBalance(assetID)
 		}
 
+		if dc.acct.isDisabled() {
+			continue // we can only attempt bond refund(if any) for disabled accounts
```

Comment:
Comment confused me. We can attempt bond refunds for **any** account. Let's make this a proper doc comment and word things a little more carfefully.
```
// For disabled account, we...
```